ENIGMAsystem/SHELL/Universal_System/Object_Tiers/planar_object.cpp

```diff
@@ -131,6 +132,83 @@ namespace enigma
     return {std::move(result), len};
   }
 
+  std::string object_planar::json_serialize() const {
+    std::string json = "{";
+
+    json += "\"object_type\":\"object_planar\",";
```

Comment:
These are good places to use a `std::stringstream`, though this leaves me wondering if it isn't better to have some kind of builder for these that could be macro'd with a single stringify. Macros are kind of unpopular, though.

Reply:
Okay, I replaced it with std::stringstream, and yeah, this is why I avoid macros.